jujugui/static/gui/src/app/store/env/bakery.js

```diff
@@ -95,13 +93,13 @@ YUI.add('juju-env-bakery', function(Y) {
             document.cookie = prefix + cfg.macaroon + ';path=/';
           }
         }
-        this.dischargeStore = cfg.dischargeStore;
-        if (!this.dischargeStore) {
-          console.error('bakery instantiated without a discharge store');
+        this.user = cfg.user;
+        if (!this.user) {
+          console.error('bakery instantiated without user authentication');
```

Comment:
Raising a new error would actually cause a break in execution, wouldn't it? Many of our tests rely on being able to create a bakery w/o this.